src/field_theory/is_alg_closed/classification.lean

```diff
@@ -0,0 +1,217 @@
+/-
+Copyright (c) 2022 Chris Hughes. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Chris Hughes
+-/
+import data.W.cardinal
+import ring_theory.algebraic_independent
+import field_theory.is_alg_closed.basic
+import field_theory.intermediate_field
+import data.polynomial.cardinal
+import data.mv_polynomial.cardinal
+import data.zmod.algebra
+/-!
+# Classification of Algebraically closed fields
+
+This file contains results related to classifying algebraically closed fields.
+
+## Main statements
+
+* `is_alg_closed.equiv_of_transcendence_basis` Two fields with the same characteristic and the same
+  cardinality of transcendence basis are isomorphic.
+* `is_alg_closed.ring_equiv_of_cardinal_eq_of_char_eq` Two uncountable algebraically closed fields
+  are isomorphic if they have the same characteristic and the same cardinality.
+-/
+universe u
+
+open_locale cardinal
+open cardinal
+
+section algebraic_closure
+
+namespace is_alg_closure
+
+variables (R L : Type u) [comm_ring R] [field L] [algebra R L]
+variables [no_zero_smul_divisors R L] [is_alg_closure R L]
+
+/--The cardinality of an algebraic closure is at most the maximum of the cardinality
+of the base ring or `ω` -/
+lemma cardinal_mk_le_max : #L ≤ max (#R) ω :=
```

Comment:
Can we increase the precedence of `#` so that we don't need all those parens as in `(#R)`.
Here it's ok, but on some of the lines below, I think it would really improve the statements.

Reply:
Sadly, I don't think there's much we can do about this. There's no precedence high enough because `max` isn't notation it's a function application.

Reply:
Ok, too bad